plugins/GSdx/GSDialog.cpp

```diff
@@ -164,12 +164,27 @@ void GSDialog::SetTextAsInt(UINT id, int i)
 	SetText(id, buff);
 }
 
-void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting>& settings, int32_t selectionValue, int32_t maxValue)
+void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting> settings, int32_t selectionValue, int32_t maxValue)
 {
+	if (settings.empty()) return;
+
 	HWND hWnd = GetDlgItem(m_hWnd, id);
+	bool currentsel_found = false;
 
 	SendMessage(hWnd, CB_RESETCONTENT, 0, 0);
 
+	for (const GSSetting i : settings)
```

Comment:
> Here some examples (because I played with the texture cache, I don't know if VS2013 support it).

I think lambda expressions were supported even at the VS2010 versions, so I _think_ it should compile.